src/app/artist-list/artist-list.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="card-body">
+  <div class="row">
+    <div class="col-md-2">Search Artist:</div>
+    <div class="col-md-4">
+      <input
+        type="text"
+        [(ngModel)]="searchstring"
+        (keyup)="searchArtist()"
+        placeholder="Search for artist"
+      />
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="column" *ngFor="let artist of artists">
+      <div class="card">
+        <img
+          [src]="artist.picture_big"
+          [title]="artist.name"
+          style="width:100%"
```

Comment:
Prefer adding CSS styling rules in the CSS file.

Reply:
@strizzwald fixed

Reply:
@strizzwald fixed